catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/TreUtilityTest.java

```diff
@@ -55,8 +55,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class TreTestUtility {
-  private static final Logger LOGGER = LoggerFactory.getLogger(TreTestUtility.class);
+public class TreUtilityTest {
```

Comment:
Unfortunate that we have a utility with tests in it.

Reply:
That is unfortunate